Audit logs (#7532)
circleci/circleci-docs: jekyll/_cci2/audit-logs.adoc

- **version:** Version of the event schema. Currently the value will always be 1. Later versions may have different values to accommodate schema changes.
- **scope:** If the target is owned by an Account in the CircleCI domain model, the account field should be filled in with the Account name and ID. This data is a JSON blob that will always contain `id` and `type` and will likely contain `name`.
- **success:** A flag to indicate if the action was successful.
- **request:** If this event was triggered by an external request this data will be populated and may be used to connect events that originate from the same external request. The format is a JSON blob containing `id` (the unique ID assigned to this request by CircleCI).

> == Request audit logs from the web app

Reviewer: I wonder if "via" might work better than "from"? (same in the para below) ... Just because I read it like you're asking the web app for something, which you kind of are. I may have over thought. Feel free to ignore.

Author: Oh I thought we were trying to avoid using the word "via" but if that's not the case, then sure.

> CircleCI server customers, as well as performance and scale cloud customers, can request audit logs from the web app. To have access to this feature, cloud customers must be organization admins.

Reviewer: ```suggestion
CircleCI server customers, as well as cloud customers on the Performance or Scale plan, can request audit logs from the web app. To have access to this feature, cloud customers must be organization admins.
```

> Navigagte to **Organization Settings > Audit Logs**. Select a date range from the dropdown, then click the **Request audit logs** button. The earliest start date is one year ago from the current day, and the lastest end date is the current day. The data available will depend on the data retention period set per organization, so the returned time period for the data could be less than the requested timeframe.

Reviewer: ```suggestion
Navigate to **Organization Settings > Audit Logs**. Select a date range from the dropdown, then click the **Request audit logs** button. The earliest start date is one year ago from the current day, and the latest end date is the current day. The data available will depend on the data retention period set per organization, so the returned time period for the data could be less than the requested timeframe.
```

> NOTE: Submitted dates are in UTC. To avoid issues arising from potential time zone differences, CircleCI adds extra time to the request. For example, if you request August 2 - August 5, the returned results will be from Aug 1 - Aug 6. The audit log request column is also displayed in UTC.

Reviewer: ```suggestion
NOTE: Submitted dates are in UTC. To avoid issues arising from potential time zone differences, CircleCI adds extra time to the request. For example, if you request August 2 - August 5, the returned results will be in the range August 1 - August 6. The audit log request column is also displayed in UTC.
```

> === Audit log status

Reviewer: Just noticed we need heading tags in this doc but we can do this in another ticket

> In the UI, a status request will show the following information:

Reviewer: Web app?

- Timeframe requested

> - User who made the request (user name or login ID)

Reviewer: ```suggestion
- User who made the request (username or login ID)
```

Reviewer: Also not sure what login ID is, is this Org ID that you can get to from the app?

- Date request was made

> - Expirary date of the request

Reviewer: ```suggestion
- Expiry date of the request
```
